CSS/CSS.sublime-syntax

```diff
@@ -1422,16 +1445,6 @@ contexts:
       pop: 1
     - include: rule-list-body
 
-  rule-list:
```

Comment:
this context was unused prior to this PR, right? I got confused for a while when I didn't see any references to it change, but that's because there wasn't any :sweat_smile:

Reply:
Yes, it already got obsolete due to a prior PR. This one was not merged at the same time due to merge conflicts in syntax tests.